### PR TITLE
cleanup code callouts for ShoppingCartService.proto

### PR DESCRIPTION
**Pull request by patriknw**
Follow up from the review of https://github.com/akka/akka-platform-guide/pull/266

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc [hunk 1]
```diff
@@ -250,7 +250,10 @@ Let's add corresponding operations as the commands in `ShoppingCartService.proto
 include::example$03-shopping-cart-service-scala/src/main/protobuf/ShoppingCartService.proto[tag=CheckoutAndGet]
 ----
 
-ifdef::todo[TODO: callout to highlight the new bool checkedOut]
+<1> Defines the `Checkout` operation.
+<2> Defines the `GetCart` operation.
+<3> <3> For simplicity, most requests share a common response, for easier evolution of an interface, separate responses are often a better choice (See xref:how-to:evolve-grpc-apis.adoc[] for details).
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc:255]:
```suggestion
<3> For simplicity, most requests share a common response, for easier evolution of an interface, separate responses are often a better choice (See xref:how-to:evolve-grpc-apis.adoc[] for details).
```


Is the `<3> <3>` some specific asciidoc feature or a mistake? (honest question)

Comment (ignasi35) [reply docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc:255]: Hmm, looks like a copy/paste error when moving the sentence from elsewhere to here.

**Review by ignasi35 — APPROVED** [review]
Minor suggestion to fix a syntax error.

LGTM otherwise

File: docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc (hunk 1 above)
Comment (patriknw) [reply docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc:255]: ouch, thanks (copy-pasta)

File: docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc [hunk 2]
```diff
@@ -250,7 +250,10 @@ Let's add corresponding operations as the commands in `ShoppingCartService.proto
 include::example$03-shopping-cart-service-scala/src/main/protobuf/ShoppingCartService.proto[tag=CheckoutAndGet]
 ----
 
-ifdef::todo[TODO: callout to highlight the new bool checkedOut]
+<1> Defines the `Checkout` operation.
+<2> Defines the `GetCart` operation.
+<3> For simplicity, most requests share a common response, for easier evolution of an interface, separate responses are often a better choice (See xref:how-to:evolve-grpc-apis.adoc[] for details).
```
Comment (octonato) [inline docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc:255]: 3 is missing in protobuf file

Comment (octonato) [reply docs-source/docs/modules/microservices-tutorial/pages/complete-entity.adoc:255]: my bad, I now see it

**Review by octonato — APPROVED** [review]

**Pull request merged**